packages/portalnetwork/src/wire/utp/PortalNetworkUtp/requestManager.ts

```diff
@@ -73,29 +74,55 @@ export class RequestManager {
         await request.init()
     }
 
-    /**
-     * Handles an incoming uTP packet.
-     * @param packetBuffer buffer containing the incoming packet
-     */
-    async handlePacket(packetBuffer: Buffer) {
-        const packet = Packet.fromBuffer(packetBuffer)
-        const request = this.lookupRequest(packet.header.connectionId)
-        if (request === undefined) {
-            this.logger.extend('HANDLE_PACKET')(`Request not found for packet - connectionId: ${packet.header.connectionId}`)
-            return
+  /**
+   * Handles an incoming uTP packet.
+   * @param packetBuffer buffer containing the incoming packet
+   */
+  async handlePacket(packetBuffer: Buffer) {
+    const packet = Packet.fromBuffer(packetBuffer)
+    const request = this.lookupRequest(packet.header.connectionId)
+    switch (request) {
+      case 'closed': {
+        if (packet.header.pType === PacketType.ST_RESET) {
+            this.logger.extend('HANDLE_PACKET')(
+              `Received RESET for closed request - connectionId: ${packet.header.connectionId}.`,
+            )
+          return
         }
-        if (packet.header.pType === PacketType.ST_SYN || packet.header.pType === PacketType.ST_RESET) {
-            await request.handleUtpPacket(packet)
-            return
+        this.logger.extend('HANDLE_PACKET')(
+          `Request closed - connectionId: ${packet.header.connectionId}.  Sending RESET to peer.`,
+        )
+        throw new Error(`REQUEST_CLOSED`)
+      }
+      case undefined: {
+        this.logger.extend('HANDLE_PACKET')(
+          `Request not found for packet - connectionId: ${packet.header.connectionId}.  Blacklisting peer.`,
+        )
+        throw new Error(`REQUEST_NOT_FOUND`)
+      }
+      default: {
+        if (
+          packet.header.pType === PacketType.ST_SYN ||
+          packet.header.pType === PacketType.ST_RESET
+        ) {
+          this.logger.extend('HANDLE_PACKET')(
+            `Processing SYN packet for Req:${packet.header.connectionId}`,
```

Comment:
```suggestion
            const packetType = packet.header.pType === PacketType.ST_SYN ? 'SYN' : 'RESET'
          this.logger.extend('HANDLE_PACKET')(
            `Processing ${packetType} packet for Req:${packet.header.connectionId}`,
```

This log message should reflect the actual packet type being reflected (since it could be either)